frontend/src/stores/QuestionStore.ts

```diff
@@ -78,6 +78,19 @@ export const useQuestionStore = defineStore('question', () => {
     localStorage.setItem('labelsbycategory', JSON.stringify(LabelsByCategory.value))
   }
 
+  function revertLabelsAtConclusion() {
+    /**
+     * This function will change all the "Niet van Toepassing" labels to "Nader te bepalen" when
+     * the back button has been clicked at the conclusion of the decision tree.
+     */
+    for (let key in LabelsByCategory.value) {
+      if (JSON.stringify(LabelsByCategory.value[key]) === JSON.stringify(['Niet van toepassing'])) {
+        LabelsByCategory.value[key] = ['Nader te bepalen']
+      }
+    }
+    localStorage.setItem('labelsbycategory', JSON.stringify(LabelsByCategory.value))
+  }
+
```

Comment:
Is it correct to revert ALL the labels? Or should only the label that was set by the conclusion be reverted?

Reply:
Well in the 6 lines before that only the "Niet van toepassing" labels are changed, and then the setItem is used to store it in the localstorage of the browser